pkg/config/dispatcher.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"github.com/spf13/pflag"
+)
+
+type DispatcherConfig struct {
+	PulseInterval            int64 `json:"pulse_interval"`
+	CheckInterval            int64 `json:"check_interval"`
+	InstanceExpirationPeriod int64 `json:"instance_expiration_period"`
+}
+
+func MewDispatcherConfig() *DispatcherConfig {
+	return &DispatcherConfig{
+		PulseInterval:            10,
+		CheckInterval:            10,
+		InstanceExpirationPeriod: 30,
+	}
+}
+
+func (c *DispatcherConfig) AddFlags(fs *pflag.FlagSet) {
+	fs.Int64Var(&c.PulseInterval, "pulse-interval", c.PulseInterval, "Interval in seconds between pulses")
```

Comment:
should be more clear, what does pulse and check interval mean? How user can choose different values among these parameters?

Reply:
added more clear description about the pulse-interval and check-interval.